### ISSUE TITLE
output equals full question

### ISSUE THREAD

**Issue by shep-kel** [AmirSasson/pardot-powerbi-connector#2]
I am using the connector you created...thanks so much for making this available. I do have a question. When using output = bulk, I am only getting the top 200 rows returned. I decided to try output = full which pulls everything. However, the first item in the xml returned is total_result. Can you tell me how I can eliminate that section in the results? Here is an example:

`<rsp stat="ok" version="1.0">
<result>
<total_results>73</total_results>
<customRedirect>
     <id>5406</id>
     <name>Some name</name>
     <url>http://wvw</url>
     <destination_url>http://cdn-www.</destination_url>
     <created_at>2017-09-11 08:57:21</created_at>
     <updated_at>2018-02-27 13:06:57</updated_at>
</customRedirect>
</result>
</rsp>`

I would like to only return the<customRedirect> sections.

Thank you for your help.

**Comment by AmirSasson** [comment]
let me understand you want to eliminate only the `total_results` element? 
are you able to ignore them or remove the column from within the powerbi query editor?
#1

**Comment by shep-kel** [comment]
yes, eliminate it before the data gets sucked into PowerBI. I am new to PowerBI so I am not sure about ignoring or removing it there. All I see when in PowerBI is that element and not the rest of the elements.

**Comment by AmirSasson** [comment]
i guess you can submit a PR that handles differently this [line](https://github.com/AmirSasson/pardot-powerbi-connector/blob/master/pardot.dataconnector/pardot.dataconnector.pq#L53)
so we can add an option for the user to decide bulk/or full retrieval of data, and if it is full treat the above line differently. let me know if u need any more clarifications.

**Comment by shep-kel** [comment]
Not sure how I would treat it...I tried Table.Skip and was unsuccessful. What do you recommend to do the job?
